databricks_cli/sdk/service.py

```diff
@@ -112,7 +112,7 @@ def submit_run(self, run_name=None, existing_cluster_id=None, new_cluster=None,
         if timeout_seconds is not None:
             _data['timeout_seconds'] = timeout_seconds
         return self.client.perform_query('POST', '/jobs/runs/submit', data=_data, headers=headers)
-
+    
```

Comment:
Seems like we are adding spaces. Is it desired?

Reply:
This file was autogenerated save for a few necessary changes, but nothing around this particular line was perturbed manually.